### PR TITLE
Collated for review

### PR DESCRIPTION
**Pull request by tshradheya**

### CODE REVIEW

**Comment by tshradheya** [comment]
@okkhoy Would be great if you could review the code quality for `/collated/main/tshradheya.md`

Made a clean(almost) PR for that.

Thanks a lot :)

File: collated/main/tshradheya.md [hunk 1]
```diff
@@ -1 +1,2700 @@
+# tshradheya
+###### \java\seedu\address\commons\events\model\DisplayPictureChangedEvent.java
+``` java
+package seedu.address.commons.events.model;
+
+import seedu.address.commons.events.BaseEvent;
+
+/**
+ * Event to trigger reading and storing of image
```
Comment (okkhoy) [inline collated/main/tshradheya.md]: better comment would be `Triggers event ...`

File: collated/main/tshradheya.md [hunk 2]
```diff
@@ -1 +1,2700 @@
...
+        this.path = path;
+        this.newPath = newPath;
+        isRead = true;
+    }
+
+    public boolean isRead() {
+        return isRead;
+    }
+
+    public void setRead(boolean b) {
```
Comment (okkhoy) [inline collated/main/tshradheya.md]: `b` ? seriously?

File: collated/main/tshradheya.md [hunk 3]
```diff
@@ -1 +1,2700 @@
...
+```
+###### \java\seedu\address\commons\events\ui\LoadPersonWebpageEvent.java
+``` java
+package seedu.address.commons.events.ui;
+
+import seedu.address.commons.events.BaseEvent;
+import seedu.address.model.person.ReadOnlyPerson;
+
+/**
+ * Event raised on 'select' command's successful execution
```
Comment (okkhoy) [inline collated/main/tshradheya.md]: may fix all header comments to follow the convention we specify

File: collated/main/tshradheya.md [hunk 4]
```diff
@@ -1 +1,2700 @@
...
+import seedu.address.commons.core.Messages;
+import seedu.address.commons.core.index.Index;
+import seedu.address.commons.events.ui.ShowDetailsEvent;
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.person.ReadOnlyPerson;
+import seedu.address.model.person.exceptions.DuplicatePersonException;
+import seedu.address.model.person.exceptions.PersonNotFoundException;
+
+/**
+ * Selects a person identified using it's last displayed index from the address book.
```
Comment (okkhoy) [inline collated/main/tshradheya.md]: selects or shows details?

File: collated/main/tshradheya.md [hunk 5]
```diff
@@ -1 +1,2700 @@
...
+        return new CommandResult(getMessageForPersonListShownSummary(model.getFilteredPersonList().size()));
+    }
+
+    @Override
+    public boolean equals(Object other) {
+        return other == this // short circuit if same object
+                || (other instanceof ViewTagCommand // instanceof handles nulls
+                && this.predicate.equals(((ViewTagCommand) other).predicate)); // state check
+    }
+
```
Comment (okkhoy) [inline collated/main/tshradheya.md:851]: errr... why so many blank lines at the end of file?

File: collated/main/tshradheya.md [hunk 6]
```diff
@@ -1 +1,2700 @@
...
+    public DisplayPictureCommand parse(String args) throws ParseException {
+        requireNonNull(args);
+        String regex = "[\\s]+";
+        String[] splitArgs = args.trim().split(regex, 2);
+
+        Index index;
+        try {
+            index = ParserUtil.parseIndex(splitArgs[0]);
+        } catch (IllegalValueException ive) {
+            throw new ParseException(String.format(MESSAGE_INVALID_COMMAND_FORMAT, DisplayPictureCommand.MESSAGE_USAGE));
```
Comment (okkhoy) [inline collated/main/tshradheya.md]: can split this line

**Review by okkhoy — COMMENTED** [review]
in general good code I would say!

**Comment by tshradheya** [comment]
Code quality improved. Merging

**Pull request merged**